features/src/main/feature/camel-features.xml

```diff
@@ -1747,6 +1728,34 @@
         <feature version='${camel.osgi.version.range}'>camel-core</feature>
         <bundle>mvn:org.apache.camel.karaf/camel-service/${project.version}</bundle>
     </feature>
+    <feature name='camel-salesforce' version='${project.version}' start-level='50'>
```

Comment:
It is not at the right place, features are alphabetically ordered

Reply:
done